docs/next/components/mdx/MDXComponents.tsx

```diff
@@ -478,6 +517,9 @@ export default {
   PlaceholderImage,
   Experimental,
   Icons,
+  ReferenceTable,
+  ReferenceTableItem,
+  // GenerateAgentToken,
```

Comment:
// intentional?

Reply:
For now! I'm going to address this before merging.